src/test/ui/abi/abi-sysv64-register-usage.rs

```diff
@@ -7,6 +7,7 @@
 // ignore-aarch64
 
 #![feature(llvm_asm)]
+#![allow(deprecated)] // llvm_asm!
```

Comment:
Shouldn't this test be modified to use `asm!` instead? This test isn't testing `llvm_asm!` itself

Reply:
Perfectly fine to leave removal to a follow-up PR (potentially the same one that removes `llvm_asm` entirely.